config/initializers/enumerable_natural_sort.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# From https://makandracards.com/makandra/9185-ruby-natural-sort-strings-with-umlauts-and-other-funny-characters
+# Copyright (c) 2012-2019 makandra GmbH, provided under the MIT License.
+module Enumerable
+  # Sorts intuitively; understands special characters and numeric ordering.
+  def natural_sort
+    natural_sort_by
+  end
+
+  # Sorts intuitively; understands special characters and numeric ordering.
+  def natural_sort_by(&stringifier)
+    sort_by do |element|
+      element = yield(element) if stringifier
+      element = element.to_s unless element.respond_to?(:to_sort_atoms)
+      element.to_sort_atoms
+    end
+  end
+
+  # Natural-sorts a list of hashes by item key.
+  def natural_sort_by_key(key = :name)
+    sort do |x, y|
+      x_value = x[key]
+      x_value = x_value.to_s unless x_value.respond_to?(:to_sort_atoms)
+
+      y_value = y[key]
+      y_value = y_value.to_s unless y_value.respond_to?(:to_sort_atoms)
+
+      x_value.to_sort_atoms <=> y_value.to_sort_atoms
+    end
+  end
+end
```

Comment:
I like the natural sort thing but I think I need to be convinced that adding all this is worth it. We are doing all kinds of sorting all over the app and it's generally just the naive kind. If there were some kind of data that had a lot of numbers at the beginning I could see it being worth it. But right now we're just sorting one of the filter lists with it, right? And we can't really use it for paginated database results. So yeah let's discuss...

Reply:
Code source again for posterity: https://makandracards.com/makandra/9185-ruby-natural-sort-strings-with-umlauts-and-other-funny-characters

We can leave this stuff in commit history but I'll remove from the codebase.